File: sdks/php/src/BillableMetrics/Types/CreateBillableMetricDtoAggregationType.php
<?php

namespace NovaBilling\BillableMetrics\Types;

enum CreateBillableMetricDtoAggregationType: string
{
    case Count = "COUNT";
    case Sum = "SUM";
    case Max = "MAX";
    case UniqueCount = "UNIQUE_COUNT";
    case Latest = "LATEST";
    case WeightedSum = "WEIGHTED_SUM";
}

File: sdks/ruby/lib/novabilling/billable_metrics/types/create_billable_metric_dto_aggregation_type.rb
# frozen_string_literal: true

module Novabilling
  module BillableMetrics
    module Types
      module CreateBillableMetricDtoAggregationType
        extend Novabilling::Internal::Types::Enum

        COUNT = "COUNT"
        SUM = "SUM"
        MAX = "MAX"
        UNIQUE_COUNT = "UNIQUE_COUNT"
        LATEST = "LATEST"
        WEIGHTED_SUM = "WEIGHTED_SUM"
      end
    end
  end
end
